composer.json

```diff
@@ -13,7 +13,7 @@
     ],
     "require": {
         "php": ">=5.3.9",
-        "doctrine/orm": "~2.2,>=2.2.3",
```

Comment:
`doctrine/orm` in version `2.2.x` requires `doctrine/common` `2.2.x` which doesn't come with the `ObjectManager::clear()` method.